### PR TITLE
update gun and babel add heroku deploy

### LINKED ISSUE #4: Client build fails
Hi. `npm run client:build` fails with the following error:
```
> gunstore@1.0.1 client:build /home/max/play/gunstore.io
> webpack

Hash: 8f3d7d29289b2e922bac
Version: webpack 3.11.0
Time: 211ms
       Asset       Size  Chunks             Chunk Names
   bundle.js    3.27 kB       0  [emitted]  main
 favicon.png    5.98 kB          [emitted]  
css/main.css    2.44 kB          [emitted]  
  index.html  929 bytes          [emitted]  
   [0] multi ./client/src/index.js 28 bytes {0} [built]
   [1] ./client/src/index.js 680 bytes {0} [built] [failed] [1 error]

ERROR in ./client/src/index.js
Module build failed: SyntaxError: Unexpected token (5:16)

  3 | import App from './components/app';
  4 | 
> 5 | ReactDOM.render(<App />, document.getElementById("app"));
    |                 ^

 @ multi ./client/src/index.js
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! gunstore@1.0.1 client:build: `webpack`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the gunstore@1.0.1 client:build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

```

Is it possible that the `.babelrc` is missing?

### PR DESCRIPTION
**Pull request by i001962**
For your consideration. Pardon all the commits.

update gun version
resolve .babel error (Fixes #4) 
add heroku deploy button

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ const cors = require('cors');
 
 const apiRoutes = require('./api/routes');
 
-const port = process.env.PORT || 3000;
+const port = process.env.PORT || 8000;
```
Comment (lmangani) [inline server.js:8]: any reason for the port change here?

File: server.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ const cors = require('cors');
 
 const apiRoutes = require('./api/routes');
 
-const port = process.env.PORT || 3000;
```
Comment (lmangani) [inline server.js]: any reason for this change since its a variable?

**Review by lmangani — CHANGES_REQUESTED** [review]

File: server.js (hunk 2 above)
Comment (i001962) [reply server.js]: Ha. No. I had forked the repo and was playing around prototyping. Then I realized I should offer the ‘fixes’ so I tried to unwind what I had done and must have forgotten this one.

File: app.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{
+    "name": "Gunstore.io",
+    "description": "A Node.js app using Express and gun",
+    "repository": "https://github.com/i001962/gunstore.io",
```
Comment (lmangani) [inline app.json]: this points to your fork and should be corrected

Comment (i001962) [reply app.json]: Oops. Will change and revert.

**Review by i001962 — COMMENTED** [review]
I think this is all you required. Let me know if you find anything else that needs adjusting.

**Pull request merged**